org.eclipse.ice.tests.integration/org.eclipse.ice.tests.integration.commands/README.md

```diff
@@ -0,0 +1,61 @@
+
+### Dependencies
+All dependencies are noted in the `pom` file, and all but one are within maven central. The only non-centralized dependency is the ICE package `org.eclipse.ice.tests.data`. To install it, perform the following commands (after cloning the ICE repositiory) so that the Commands package can build successfully:
```

Comment:
This README should be updated, but I'll add that as an issue since I should deal with it.